internal/rpc/streamproxy_test.go

```diff
@@ -14,19 +14,15 @@ import (
 	"github.com/canonical/jimm/v3/internal/rpc"
 )
 
-func streamEcho(c *websocket.Conn, stopped *bool) error {
-	for {
-		msg := make(map[string]interface{})
-		if *stopped {
-			return errors.New("stopped")
-		}
-		if err := c.ReadJSON(&msg); err != nil {
-			return err
-		}
-		if err := c.WriteJSON(msg); err != nil {
-			return err
-		}
+func streamEcho(c *websocket.Conn) error {
```

Comment:
so, this will echo a single message.. the function name seems wrong, that's my comment :)

Reply:
updated the name to a more explicit